spec.html

```diff
@@ -20632,6 +21226,218 @@ <h1>Static Semantics: PrototypePropertyNameList</h1>
+
+        AsyncFunctionDeclaration : `async` `function` `(` FormalParameters `)` `{` AsyncFunctionBody `}`
+
+        AsyncFunctionExpression : `async` `function` `(` FormalParameters `)` `{` AsyncFunctionBody `}`
+
+        AsyncFunctionExpression : `async` `function` BindingIdentifier `(` FormalParameters `)` `{` AsyncFunctionBody `}`
+      </emu-grammar>
+      <emu-alg>
+        1. Return *false*.
+      </emu-alg>
+
+      <p>For all other grammatical productions, recurse on all nonterminals. If any piece returns *true*, then return *true*. Otherwise return *false*.</p>
+    </emu-clause>
+
+    <emu-clause id="sec-class-definitions-runtime-semantics-classfielddefinitionevaluation" type="sdo" aoid="ClassFieldDefinitionEvaluation">
+      <h1>Runtime Semantics: ClassFieldDefinitionEvaluation</h1>
+
+      <p>With parameter _homeObject_.</p>
+
+      <emu-grammar>
+        FieldDefinition : ClassElementName Initializer?
+      </emu-grammar>
+      <emu-alg>
+        1. Let _name_ be the result of evaluating |ClassElementName|.
+        1. ReturnIfAbrupt(_name_).
+        1. If |Initializer_opt| is present, then
+          1. Let _lex_ be the LexicalEnvironment of the running execution context.
+          1. Let _formalParameterList_ be an instance of the production <emu-grammar>FormalParameters : [empty]</emu-grammar>.
+          1. Let _privateScope_ be the running execution context's PrivateEnvironment.
+          1. Let _initializer_ be ! OrdinaryFunctionCreate(~Method~, _formalParameterList_, |Initializer|, _lex_, _privateScope_).
+          1. Perform MakeMethod(_initializer_, _homeObject_).
+          1. Set _initializer_.[[ClassFieldInitializer]] to *true*.
+          1. Let _isAnonymousFunctionDefinition_ be IsAnonymousFunctionDefinition(|Initializer|).
+        1. Else,
+          1. Let _initializer_ be ~empty~.
+          1. Let _isAnonymousFunctionDefinition_ be *false*.
+        1. Return the ClassFieldDefinition Record { [[Name]]: _name_, [[Initializer]]: _initializer_, [[IsAnonymousFunctionDefinition]]: _isAnonymousFunctionDefinition_ }.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-class-definitions-static-semantics-classelementevaluation" type="sdo" aoid="ClassElementEvaluation">
+      <h1>Runtime Semantics: ClassElementEvaluation</h1>
+      <p>With parameters _object_ and _enumerable_.</p>
+
+      <emu-grammar>
+        ClassElement : FieldDefinition `;`
+
+        ClassElement : `static` FieldDefinition `;`
+      </emu-grammar>
+      <emu-alg>
+        1. Return ClassFieldDefinitionEvaluation of |FieldDefinition| with argument _object_.
+      </emu-alg>
+
+      <emu-grammar>
+        ClassElement : MethodDefinition
+
+        ClassElement : `static` MethodDefinition
+      </emu-grammar>
+      <emu-alg>
+        1. Return PropertyDefinitionEvaluation of |MethodDefinition| with arguments _object_ and _enumerable_.
```

Comment:
Should `PropertyDefinitionEvaluation ` be `MethodDefinitionEvaluation`? The MethodDefinition does not define `PropertyDefinitionEvaluation` anyway.

Reply:
I went ahead and changed it in https://github.com/tc39/ecma262/pull/1668/commits/ba31b69717a2bdb6a3b4b1863e6e71ba78491ac8.